bin/improver-blend-adjacent-points

```diff
@@ -55,6 +55,13 @@ def main():
                         metavar='COORDINATE_TO_BLEND_OVER',
                         help='The coordinate over which the blending '
                              'will be applied.')
+    parser.add_argument('central_point', metavar='CENTRAL_POINT', type=float,
+                        help='Central point at which the output from the '
+                             'triangular weighted blending will be '
+                             'calculated. This should be in the units of the '
+                             'parameter_unit that is passed in.'
+                             'This value should be a point on the '
+                             'coordinate for blending over.')
```

Comment:
After discussing with @MoseleyS, as the intention is to specify a `forecast_period` as the `central_point`, rather than an ISO-8601 (cylc-format) time, at the moment, having the central_point specified as a float is satisfactory.